### DISCUSSION TITLE
Latest release behind head of main branch

### DISCUSSION THREAD

**Discussion by meganvw** [long2ice/fastapi-cache#429] [General]
Hi, I see that the last published release v0.2.1 was published after the latest changes on main.

Specifically I'm looking to read the "HIT" or "MISS" value added to cache_status_header, which seems to have been added after the latest release.

Are you planning to do another release by any chance?

Thanks!

**Comment by CharlesPerrotMinot** [comment]
Doing a release would also update dependencies, would be pretty useful! @long2ice 
Don't hesitate if you need support here (be it because of time constraints or any reason), happy to help!
